WordPress/Classes/ViewRelated/Cells/BorderedButtonTableViewCell.swift

```diff
@@ -26,6 +26,44 @@ class BorderedButtonTableViewCell: UITableViewCell {
     private var highlightedColor = Defaults.highlightedColor
     private var borderColor = Defaults.normalColor
 
+    // Toggles the loading state of the cell.
+    var isLoading: Bool = false {
```

Comment:
Even though I'm using this to indicate deleting, since this is a generic button I left this generic as well.